README.md

```diff
@@ -25,6 +25,22 @@ For instance, use the following command to use the `master` branch of Symfony:
 STABILITY=dev docker-compose up --build
 ```
 
+## Build and running behind a proxy
+
+1. Uncomment in the Dockerfile the following line
+```bash
+RUN pear config-set http_proxy yourproxy
+```
+2. Replace "yourproxy" by your proxy
+3. Build
+```bash
+docker-compose build --build-arg http_proxy="yourproxy" --build-arg https_proxy="yourproxy"
+```
+4. Up
+```bash
+docker-compose up
```

Comment:
Usually I use -d too but since in the doc. it was already using docker-compose up without the -d I thought it will be better to keep the same behaviour.